backend/inspirehep/records/api/literature.py

```diff
@@ -507,6 +508,7 @@ def add_file(
             new_key = hash_data(file_data)
             mimetype = magic.from_buffer(file_data, mime=True)
             file_data = BytesIO(file_data)
+            is_scanned = is_document_scanned(file_data)
```

Comment:
can we somehow narrow down the usage of this? Can we get some performance benchmark for files with many pages?

Reply:
1) Do you have any suggestions?
2) maybe we can add a prometheus metric

Reply:
@drjova